front/app/components/ProjectFolders/ProjectFolderSharingModal/index.tsx

```diff
@@ -26,85 +25,95 @@ interface Props {
   close: () => void;
 }
 
-const ProjectFolderSharingModal = memo<
-  Props & WrappedComponentProps & InjectedLocalized
->(({ projectFolderId, className, opened, close, intl: { formatMessage } }) => {
-  const authUser = useAuthUser();
-  const projectFolder = useProjectFolder({ projectFolderId });
+const ProjectFolderSharingModal = memo<Props & InjectedLocalized>(
```

Comment:
Maybe we should also remove InjecttedLocalized and use `useLocalize` instead, if we are touching the component anyway?